samples/local_attestation/enclave_a/CMakeLists.txt

```diff
@@ -5,8 +5,8 @@
 add_custom_command(
   OUTPUT enclave_b_pubkey.h
   DEPENDS public_key_b ${CMAKE_SOURCE_DIR}/gen_pubkey_header.sh
-  COMMAND ${OE_BASH} ${CMAKE_SOURCE_DIR}/gen_pubkey_header.sh
-          enclave_b_pubkey.h ${CMAKE_BINARY_DIR}/enclave_b/public_b.pem)
+  COMMAND ${CMAKE_SOURCE_DIR}/gen_pubkey_header.sh enclave_b_pubkey.h
```

Comment:
The use of `OE_BASH` was specifically to work around invoking bash scripts on Windows, and it's interesting that it was not uniformly applied to both samples. I believe that we do run the samples tests on Windows, so if this is passing, it's apparently not an issue. 

@BRMcLaren in case I'm wrong about samples being run on Windows.

Reply:
Another case where one sample did it one way and the other sample did it the other way.   My assumption was the same as yours.  If both samples passed testing already, then it obviously wasn't needed and could be removed.   If it didn't pass testing, well then at least one sample was already broken before this PR.